src/lib/util.js

```diff
@@ -69,3 +69,11 @@ exports.formatMs = function(ms) {
 exports.removeTrailingSlash = function(str = '') {
   return str.endsWith('/') ? str.substr(0, str.length - 1) : str;
 };
+
+exports.streamToString = async function streamToString(readable) {
+  let result = '';
+  for await (const chunk of readable) {
```

Comment:
Unfortunately, `for await` loop not available in Node `v8.12`, that is our target node version at the moment. Please use `Promise`. It's a bit unfortunate that we don't have a `CI` setup to run the test suits. Otherwise, you could've caught this incompatibility issue a bit earlier.

![image](https://user-images.githubusercontent.com/12813750/81591499-24562380-93da-11ea-9a6d-24dead32cec2.png)

I've already created a ticket to set up the CI. https://github.com/m-sureshraj/jenni/issues/32